schemas/blocks/calculated_summary.json

```diff
@@ -24,26 +24,55 @@
         "$ref": "https://eq.ons.gov.uk/common_definitions.json#/skip_conditions"
       },
       "calculation": {
-        "type": "object",
-        "properties": {
-          "calculation_type": {
-            "type": "string",
-            "enum": ["sum"]
-          },
-          "answers_to_calculate": {
-            "type": "array",
-            "minItems": 2,
-            "items": {
-              "$ref": "https://eq.ons.gov.uk/common_definitions.json#/identifier"
+        "oneOf": [
+          {
+            "type": "object",
+            "properties": {
+              "calculation_type": {
+                "type": "string",
+                "enum": ["sum"]
+              },
+              "answers_to_calculate": {
+                "type": "array",
+                "minItems": 2,
+                "items": {
+                  "$ref": "https://eq.ons.gov.uk/common_definitions.json#/identifier"
+                },
+                "description": "The id of the answers from which to do calculated_type to"
+              },
+              "title": {
+                "$ref": "https://eq.ons.gov.uk/string_interpolation/definitions.json#/string_with_placeholders"
+              }
             },
-            "description": "The id of the answers from which to do calculated_type to"
+            "additionalProperties": false,
+            "required": ["calculation_type", "answers_to_calculate", "title"]
           },
-          "title": {
-            "$ref": "https://eq.ons.gov.uk/string_interpolation/definitions.json#/string_with_placeholders"
+          {
+            "type": "object",
+            "properties": {
+              "operation": {
+                "type": "object",
+                "properties": {
+                  "+": {
+                    "type": "array",
+                    "items": [
+                      {
+                        "$ref": "https://eq.ons.gov.uk/value_sources.json#/value_source_for_calculations",
+                        "description": "A value source to validate against that resolves to a number."
+                      }
+                    ]
+                  }
```

Comment:
- This should be introduced as a new numeric operator as part of rules. We currently have boolean and value operators. This is just a new one.

- Rules should also be extended and tested
- The numeric operator should allow pretty much any source, for answers we need to check it is numeric. There is code validation to do such things already which should be extended.
- It needs to allow recursion of other numerical operators (currently only addition)

Think of it as you need to add a new `+` in general than calculated summary specifically. We just happen to reuse it for calculated summaries.

Reply:
Added the new numeric "+" operator 👍